{"name": "Проверка, что незарегистрированный пользователь не может войти в сиситему, появление ошибки", "status": "passed", "start": 1758314307055, "stop": 1758314307146, "uuid": "18489208-396d-4631-b13d-d5dca2ac97e2", "historyId": "9844aacfc95dd82a54b24d124acdc0f7", "testCaseId": "9844aacfc95dd82a54b24d124acdc0f7", "fullName": "tests.test_courier_login.TestCourierLogin#test_unregistered_courier_cannot_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-U7O3CJE"}, {"name": "thread", "value": "7372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}], "titlePath": ["tests", "test_courier_login.py", "TestCourierLogin"]}